migrations/initdb-jobs.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}jobs`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.createTable(table, {
+                id: {
+                    allowNull: false,
+                    autoIncrement: true,
+                    primaryKey: true,
+                    type: Sequelize.INTEGER
+                },
+                name: {
+                    type: Sequelize.STRING(110)
+                },
+                permutations: {
+                    type: Sequelize.TEXT
+                },
+                description: {
+                    type: Sequelize.STRING(100)
+                },
+                pipelineId: {
+                    type: Sequelize.DOUBLE
+                },
+                state: {
+                    type: Sequelize.STRING(10)
+                },
+                archived: {
+                    type: Sequelize.BOOLEAN
+                },
+                prParentJobId: {
+                    type: Sequelize.DOUBLE
+                },
+                stateChanger: {
+                    type: Sequelize.STRING(128)
+                },
+                stateChangeTime: {
+                    type: Sequelize.TEXT
+                },
+                stateChangeMessage: {
+                    type: Sequelize.STRING(512)
+                }
+            }, { transaction }
+            );
+
+            await queryInterface.addConstraint(table, ['name', 'pipelineId'],
+                {
+                    name: `${table}_name_pipelineId_key`,
+                    type: 'unique',
+                    transaction
+                }
+            );
+
+            await queryInterface.addIndex(table, ['pipelineId'],
+                {
+                    name: `${table}_pipeline_id`,
+                    transaction
+                }
+            );
+
+            await queryInterface.addIndex(table, ['state'],
+                {
+                    name: `${table}_state`,
+                    transaction
+                }
+            );
```

Comment:
Index is missing:
> KEY `jobs_pipeline_id_state` (`pipelineId`,`state`)
defined at this line:
https://github.com/screwdriver-cd/data-schema/blob/master/models/job.js#L144

Reply:
This index is not available in database. Do you have this on your database?

Reply:
Yes, we have the index in our mysql 5.7. The definition is as I post.
It seems that postgresql can handle multicolumn indexes. Does sequelizejs handle alter query properly?
http://www.postgresqltutorial.com/postgresql-indexes/postgresql-multicolumn-indexes/